jax/_src/interpreters/partial_eval.py

```diff
@@ -2663,3 +2675,24 @@ def _linearize_of_pmap_hack(f: lu.WrappedFun, jaxpr, consts) -> tuple[Jaxpr, lis
     _, jaxpr = f.f.closure
     return convert_constvars_jaxpr(jaxpr), []
   return jaxpr, consts
+
+
+def lower_jaxpr(hi_jaxpr):
```

Comment:
nit: should this be cached since it returns a ClosedJaxpr

Reply:
Done, thanks